src/app/auth/duck/errorActions.ts

```diff
@@ -0,0 +1,13 @@
+const certErrorOccurred = (failedUrl: string) =>
```

Comment:
Not sure about this split, I don't think there are enough auth actions to warrant this. Also, if you're going to split off actions into their own files, the pattern is to turn `actions` into a directory, ex: `src/app/auth/duck/actions/*` with an `index.ts` file that imports the files that you have split up. That way a consumer can use the import path `import { Creators } from './src/app/auth/duck/actions'`. It will load the index.ts and run that code. You might have to re-export things from there. It makes the directory appear as if it's a single module, and allows us to internally organize it into separate files.